lib/gui/components/update-notifier/services/update-notifier.js

```diff
@@ -131,24 +62,28 @@ module.exports = function($http, $q, ModalService, UPDATE_NOTIFIER_SLEEP_DAYS, M
    * @function
    * @public
    *
+   * @param {String} version - version
+   * @param {Object} options - options
+   * @param {Boolean} [options.allowSleepUpdateCheck=true] - allow sleeping the update check
    * @returns {Promise}
    *
    * @example
-   * UpdateNotifierService.notify();
+   * UpdateNotifierService.notify('1.0.0-beta.16', {
+   *   currentVersion: '1.0.0-beta.15'
+   * });
    */
-  this.notify = () => {
-    return this.getLatestVersion().then((version) => {
-      return ModalService.open({
-        template: './components/update-notifier/templates/update-notifier-modal.tpl.html',
-        controller: 'UpdateNotifierController as modal',
-        size: 'update-notifier',
-        resolve: {
-          options: _.constant({
-            version
-          })
-        }
-      }).result;
-    });
+  this.notify = (version, options) => {
```

Comment:
Its not really necessary given that we're using `_.get`, which copes with undefined objects, but lets add it for clarity purposes.